test/regressions/index.test.js

```diff
@@ -197,7 +197,7 @@ async function main() {
 
       return new Promise((resolve, reject) => {
         // See https://ffmpeg.org/ffmpeg-devices.html#x11grab
-        const args = `-y -f x11grab -framerate 1 -video_size 460x400 -i :99.0+90,81 -vframes 1 ${screenshotPath}`;
+        const args = `-y -f x11grab -framerate 1 -video_size 460x400 -i :99.0+90,85 -vframes 1 ${screenshotPath}`;
```

Comment:
The print window offset has changed in Chrome, and after updating the value there are no visual changes reported 🙂

Reply:
Nice! 👏 
Wasn't aware of it, thank you for looking into it. 🙏